src/CabalFmt/Glob.hs

```diff
@@ -1,8 +1,15 @@
+{-# LANGUAGE CPP #-}
+
 module CabalFmt.Glob where
 
 import Data.List             (isInfixOf)
 import Data.List.NonEmpty    (NonEmpty (..))
-import System.FilePath.Posix (splitDirectories)
+
+#ifdef mingw32_HOST_OS
+import System.FilePath.Windows (splitDirectories)
```

Comment:
globs in cabal-fmt pragmas should always be system independent, which means they should use `/` separators.